### DISCUSSION TITLE
Automatic prune/cleanup after upgrade

### DISCUSSION THREAD

**Discussion by madiele** [louislam/dockge#192] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

It's pretty cool to be able to upgrade from the UI, though since there is no way to do a `docker system prune -a` right from the GUI right now you end-up with lots of garbage and having to use ssh anyway. This is a problem especially on cheap servers low on storage like the base tier droplet on digital ocean.

### ✔️ Solution

would be cool to have the option for something like "upgrade and prune" or maybe something more smart that will only prune the stuff associated to what was upgraded

### ❓ Alternatives

`docker system prune -a --force` inside the dockge cli will work, but with no command history is still kinda uncomfortable

**Comment by LilTrublMakr** [comment]
+1 for this. Looking in to an Ansible playbook to do this currently but would be really nice to be able to do it via the GUI.

**Reply by madiele** [reply]
BTW `docker system prune -a --force` will work even inside the dockge command line, which at least something
